ivy_tests/test_ivy/test_frontends/test_tensorflow/test_raw_ops.py

```diff
@@ -3058,3 +3058,86 @@ def test_tensorflow_Elu(
         features=x[0],
         name=name,
     )
+
+
+@st.composite
+def _conv2d_helper(draw):
+    input_dtype, \
+        x, \
+        filters, \
+        strides, \
+        dilations, \
+        data_format, \
+        padding = _x_and_filters(draw)
```

Comment:
I think this should be `draw(_x_and_filters())`.

Reply:
And don't forget to pass arguments to `_x_and_filters` as well if they're needed.